pkg/cmd/openshift-operators/webconsole-operator/v310_00/deployment.go

```diff
@@ -0,0 +1,71 @@
+package v310_00
+
+const DeploymentYaml = `apiVersion: apps/v1
+kind: Deployment
+metadata:
+  namespace: openshift-web-console
+  name: webconsole
+  labels:
+    app: openshift-web-console
+    webconsole: "true"
+spec:
+  strategy:
+    type: Recreate
```

Comment:
is there reason this is Recreate and not Rollout ?

Reply:
It was discussed some time ago, web-console switched to recreate some time ago, iirc.